public/video-ui/styles/layout/_video.scss

```diff
@@ -67,17 +59,21 @@
   width: 300px;
 }
 
+.video__detailbox__header__container {
+  display: flex;
+  justify-content: space-between;
+}
+
 .video__detailbox {
+  width: 23vw;
```

Comment:
veeery magical. As this is for layout, there's probably a more mathematical approach to this.